frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateSelectStep.tsx

```diff
@@ -0,0 +1,46 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import { useActions } from 'kea'
+import { useEffect } from 'react'
+import { DashboardTemplateChooser } from 'scenes/dashboard/DashboardTemplateChooser'
+import { newDashboardLogic } from 'scenes/dashboard/newDashboardLogic'
+
+import { onboardingLogic, OnboardingStepKey } from '../onboardingLogic'
+import { OnboardingStep } from '../OnboardingStep'
+
+export const OnboardingDashboardTemplateSelectStep = ({
+    stepKey = OnboardingStepKey.DASHBOARD_TEMPLATE,
+}: {
+    stepKey?: OnboardingStepKey
+}): JSX.Element => {
+    const { goToNextStep } = useActions(onboardingLogic)
+    const { clearActiveDashboardTemplate } = useActions(newDashboardLogic)
+
+    // TODO: this is hacky, find a better way to clear the active template when coming back to this screen
+    useEffect(() => {
+        clearActiveDashboardTemplate()
+    }, [])
```

Comment:
You could do something like this that clears it on unmount
```suggestion
    useEffect(() => {
        return () => {
            clearActiveDashboardTemplate()
        }
    }, [])
```

Reply:
I also tried this, but as soon as this component unmounted (to go to the variable setup step) it would clear the template, and redirect back to this page. So clearing it on mount is the most streamlined solution so far. I could maybe put it in the logic `onMount` but the logic also mounts on the next step. I will play with it in the next PR.